src/sentry/integrations/github/integration.py

```diff
@@ -160,8 +146,6 @@ class GitHubIntegrationProvider(IntegrationProvider):
     integration_cls = GitHubIntegration
     features = frozenset([IntegrationFeatures.COMMITS, IntegrationFeatures.ISSUE_BASIC])
 
-    can_disable = True
```

Comment:
I'd explicitly set `can_disable` to false and leave comment describing the user issue.
Also the comment above `IntegrationProvider.can_disable` doesn't really describe what disabling does.

Reply:
I agree it's a little confusing because technically the integration can still be set to disabled if the user uninstalls from GH first. But we use this attribute for whether or not we show the disable dialog text. 

I think we should really be documenting the install and uninstall flows for our integrations and related data (i.e. repos, commits, alert rules)

Reply:
@MeredithAnya @mgaeta I added a new comment in the base class what `can_disable` actually does